synchronizer/connectors/jira.py

```diff
@@ -62,6 +63,21 @@ def export_worklogs(self, worklogs):
                 print(err)
                 raise self.ExportException(i)
 
+    def validate_issue(self, issue_id):
+        """
+        Check existing issue id in target resource

```

Comment:
`Check if issue exists in JIRA.`

Also add information about what values function can return.